forte/data/ontology/core.py

```diff
@@ -601,8 +601,8 @@ def __iter__(self) -> Iterator[KeyType]:
 class FNdArray:
     """
     FNdArray is a wrapper of a NumPy array that stores shape and data type
-    of the array when they are specified. Only when both shape and data type
-    are provided, will FNdArray initializes an empty array accordingly.
+    of the array if they are specified. Only when both shape and data type
+    are provided, will FNdArray initialize an empty array accordingly.
```

Comment:
So is `empty` accurate here, better refer to the numpy documentation to ensure users expect the actual content of the array correctly: https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html

Reply:
agree